src/main/scala/util/package.scala

```diff
@@ -234,23 +234,55 @@ package object util {
     map.view.map({ case (k, vs) => k -> vs.toList }).toList
   }
 
+/** provides operators useful for working with bidirectional [[Bundle]]s
+  * 
+  * In terms of [[Flipped]] with a producer 'p' and 'consumer' c:
+  * c :<= p // means drive all unflipped fields of 'c' from 'p' (e.g.: valid/bits)  
+  * c :=> p // means drive all flipped fields of 'c' from 'p' (e.g.: ready)  
```

Comment:
this one actually means
` * c :=> p // means drive all flipped fields of 'p' from 'c' (e.g.: ready) ` , e.g.: p.ready := c.ready ?

Reply:
so should this say p :=> c for a more consistent example?

Reply:
My understanding is "no". 

Rule #1 for this set of operator is "consumer always on LHS". (It is sort of directional operator from the producer-consumer's viewpoint. not sure of the best way to express this).

p:=> c means that the consumer's ready (input) is driven by the producer's ready (output), which is wrong.

Reply:
You are correct. Consumer is always on the left-hand-side. The comment should have been:
```
c :=> p // means drive all flipped fields of 'p' from 'c' (e.g.: p.ready := c.ready)
```